src/raftstore/store/peer.rs

```diff
@@ -513,6 +532,21 @@ impl Peer {
         Ok(())
     }
 
+    fn use_local_read(&self, req: &RaftCmdRequest) -> bool {
```

Comment:
IMO, `use_local_read` seems that we will use local read in this function, but this function is just to check whether we need local read or not.